src/main/java/edu/kis/powp/jobs2d/drivers/adapter/MyLineDrawerAdapter.java

```diff
@@ -0,0 +1,37 @@
+package edu.kis.powp.jobs2d.drivers.adapter;
+
+import edu.kis.legacy.drawer.panel.DrawPanelController;
+import edu.kis.legacy.drawer.shape.ILine;
+import edu.kis.legacy.drawer.shape.LineFactory;
+import edu.kis.powp.jobs2d.Job2dDriver;
+
+public class MyLineDrawerAdapter extends DrawPanelController implements Job2dDriver {
```

Comment:
'My' prefix is a bad practice.
 The name is better but it lacks a driver aspect.